packages/atomic/src/components/atomic-result/atomic-result.pcss

```diff
@@ -224,6 +423,14 @@ atomic-result-section-bottom-metadata {
   color: var(--atomic-neutral-dark);
 }
 
-section atomic-result-quickview {
-  display: inline-block;
+atomic-result-section-visual,
+atomic-result-section-badges,
+atomic-result-section-actions,
+atomic-result-section-title,
+atomic-result-section-title-metadata,
+atomic-result-section-emphasized,
+atomic-result-section-excerpt,
+atomic-result-section-bottom-metadata {
+  overflow: hidden;
+  text-overflow: ellipsis;
 }
```

Comment:
One result in particular contained a long name with no spaces, which showed that text could overflow from a section.